examples/print-repository-velocity.pl

```diff
@@ -0,0 +1,20 @@
+use strict;
+use warnings;
+
+use GitHub::MergeVelocity;
+
+my $mv = GitHub::MergeVelocity->new(
+    url => [
+        'neilbowers/PAUSE-Permissions',
+        'https://github.com/oalders/html-restrict/issues',
+    ]
+);
+
+my @repos = map { $mv->_repository_for_url($_) } $mv->_report_urls;
```

Comment:
We should probably make `_repository_for_url` and `_report_urls` public if we're going to do this. I try to make the majority of the API private by default unless there's a reason for it to be public. Doing the above demonstrates that those methods should probably be public.